src/targets/gpu/kernels/include/migraphx/kernels/compute_attention_probabilities.hpp

```diff
@@ -85,16 +85,18 @@ calculate_attention_probs(AttnProbs attention_probs, // output buffer with size
 }
 
 template <class Output,
-          class Query,
+          class Passthrough,
           class PresentKey,
           class PresentValue,
           class SeqLensK,
+          class Query,
           class Params>
 __device__ void compute_attention_probabilities(Output output,
-                                                Query query,
+                                                Passthrough, // Used for shape info and graph ordering only
```

Comment:
I dont really follow how these parameters match what is done in prefuse_ops. You dont really change the first parameter but somehow this is changed here. Is the query input supposed to be the concat or the `rotary_qkv`?